std/meta.d

```diff
@@ -986,24 +985,23 @@ template aliasSeqOf(alias range)
         {
             alias aliasSeqOf = AliasSeq!();
         }
-        else static if (range.length == 1)
-        {
-            alias aliasSeqOf = AliasSeq!(range[0]);
-        }
         else
         {
-            alias aliasSeqOf = AliasSeq!(aliasSeqOf!(range[0 .. $/2]), aliasSeqOf!(range[$/2 .. $]));
+            alias aliasSeqOf = AliasSeq!(range[0], aliasSeqOf!(range[1 .. $]));
```

Comment:
> A range, which does not contain significant count of identical values.

I can understand all changes except this divide and conquer --> linear change, seems like net negative to me